TodoReview.py

```diff
@@ -150,6 +150,8 @@ def run(self, edit, **args):
 		window = self.view.window()
 		paths = args.get('paths', None)
 		settings = Settings(self.view, args.get('settings', False))
+		if not paths and not settings.get('current_file', False):
```

Comment:
I think we can drop the `if not paths` on this. My main train of though is this argument will only be `True` if it is bound to a key, so the user has already explicitly asked for the current file only. Let me know if you disagree.